scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/task/TaskInfoImpl.java

```diff
@@ -198,6 +203,14 @@ public void setStartTime(long startTime) {
         this.startTime = startTime;
     }
 
+    public long getInErrorTime() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)